internal/satellite/module/sender/sender.go

```diff
@@ -216,3 +217,12 @@ func (s *Sender) consume(batch *buffer.BatchBuffer) {
 func (s *Sender) InputDataChannel() chan<- *event.OutputEventContext {
 	return s.input
 }
+
+func (s *Sender) SyncProcess(data *v1.SniffData) (*v1.SniffData, error) {
```

Comment:
How about adding a new interface in forwarder API called supportedSyncInvoke to select the sync forwarder